Loop/Views/BolusEntryView.swift

```diff
@@ -334,6 +334,11 @@ struct BolusEntryView: View {
                 title: Text("No Recent Glucose Data", comment: "Title for bolus screen notice when glucose data is missing or stale"),
                 caption: Text("Enter a blood glucose from a meter for a recommended bolus amount.", comment: "Caption for bolus screen notice when glucose data is missing or stale")
             )
+        case .futureGlucoseData:
+            return WarningView(
+                title: Text("Glucose in Future", comment: "Title for bolus screen notice when glucose data is in the future"),
```

Comment:
Title: "Invalid  Future Glucose"
Caption: "Check your device time and/or remove any invalid data from Apple Health"

Reply:
Done. Will push changes soon.